burn-compute/tests/dummy/compute.rs

```diff
@@ -30,3 +31,16 @@ pub fn client(device: &DummyDevice) -> DummyClient {
         ComputeClient::new(channel, tuner)
     })
 }
+
+pub fn client2(device: &DummyDevice) -> DummyClient {
+    COMPUTE2.client(device, || {
+        let storage = BytesStorage::default();
+        let memory_management =
+            SimpleMemoryManagement::new(storage, DeallocStrategy::Never, SliceStrategy::Never);
+        let server = DummyServer::new(memory_management);
+        let channel = MutexComputeChannel::new(server);
+        let tuner = Arc::new(Mutex::new(Tuner::new()));
+
+        ComputeClient::new(channel, tuner)
+    })
+}
```

Comment:
If you need your own instance of `COMPUTE` for the tests, I would suggest to create this in the test file you need it with a better name: `persistent_cache_test_client`.